src/cps/utils.cpp

```diff
@@ -21,6 +23,17 @@ namespace cps::utils {
         return out;
     }
 
+    std::string_view trim(std::string_view input) {
```

Comment:
It's semi-bonkers this isn't a thing in `fmt` or the standard library.

I added a Beman Project issue for someone to think about this for standard C++:
https://github.com/beman-project/beman/issues/51